crates/stc_ts_env/src/lib.rs

```diff
@@ -222,6 +223,7 @@ impl From<&CompilerOptions> for Rule {
             allow_unused_labels: v.allow_unused_labels.unwrap_or_default(),
             no_fallthrough_cases_in_switch: v.no_fallthrough_cases_in_switch.unwrap_or_default(),
             no_implicit_returns: v.no_implicit_returns.unwrap_or_default(),
+            no_unchecked_indexed_access: false,
```

Comment:
i can add this feature to [tsconfig](https://github.com/drivasperez/tsconfig) directory(I'm not sure the maintainer will response). Do you keep using this crate??

Reply:
I think vendoring it would worth the effort.